rankmanager.lua

```diff
@@ -0,0 +1,178 @@
+--[[
+
+    Overwatch Rank Manager for Helix
+
+    Installation:
+    
+    1. Edit the self.rankTable variable within PLUGIN:OnLoaded() to fit your rank needs.
+    2. Place this Lua file into your schema plugin directory.
+    3. Enable allowRankManager in your server config
+
+]]
+
+local PLUGIN = PLUGIN
+
+PLUGIN.name = "Overwatch Rank Manager"
+PLUGIN.author = "Gary Tate"
+PLUGIN.description = "Allows Overwatch to manage ranks."
+PLUGIN.rankTable = PLUGIN.rankTable or {}
+
+ix.lang.AddTable("english", {
+    allowRankManager = "Allow Overwatch Rank Manager",
+    cmdRankDemote = "Demote an active Overwatch functionary.",
+    cmdRankPromote = "Promote an active Overwatch functionary.",
+    cRankConfigDisabled = "Rank Management is currently disabled.",
+    cRankPromotion = "%s has promoted %s to %s.",
+    cRankDemotion = "%s has demoted %s to %s.",
+    cRankMaxRank = "%s is already the maximum rank.",
+    cRankMinRank = "%s is already the minimum rank.",
+    cRankInvalidRank = "%s is an invalid rank, cannot promote/demote.",
+    cRankInvalidFaction = "%s is not in a valid faction.",
+    cRankInvalidInput = "%s is not a valid rank."
+})
+
+ix.config.Add("allowRankManager", false, "Enable/Disable Overwatch ability to manage ranks.", nil, {

```

Comment:
Is there much point in having a toggle for this when plugins themselves can be disabled? Not an issue just unsure if it's really worth having